corehq/apps/users/models.py

```diff
@@ -1623,13 +1630,34 @@ def sql_location(self):
                 pass
         return None
 
+    @property
+    def sql_locations(self):
+        from corehq.apps.locations.models import SQLLocation
+        if self.assigned_location_ids:
+            try:
+                return SQLLocation.objects.filter(location_id__in=self.assigned_location_ids)
+            except SQLLocation.DoesNotExist:
+                pass
+        return []
```

Comment:
See: 
https://github.com/dimagi/commcare-hq/pull/14019#discussion_r89429985
I think this should be `SQLLocation.objects.none()` so this can be chainable with other queryset methods.